numpy/lib/function_base.py

```diff
@@ -2965,7 +2957,7 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
 
     >>> x = [-2.1, -1,  4.3]
     >>> y = [3,  1.1,  0.12]
-    >>> X = np.vstack((x,y))
+    >>> X = np.concatenate([np.atleast_2d(m) for m in (x,y)], axis=0)
```

Comment:
Ah, I see. But this is the docs for `cov`, which is not aiming to show you how to stack matrices and vectors, so I think simplicity (`stack`) is best here.

As an aside, `np.atleast_2d(x, y)` is a thing that saves a list comprehension